static/js/components/ClassificationForm.jsx

```diff
@@ -201,7 +201,7 @@ const ClassificationForm = ({ obj_id, taxonomyList }) => {
     );
   };
   CustomGroupsWidget.propTypes = {
-    value: PropTypes.string.isRequired,
+    value: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
Nice!  Good catch - actually just made a story about this yesterday #1593. Thanks for the fix!